docs/build-from-source.md

```diff
@@ -81,11 +81,11 @@ These values are then defacto constants, which means code optimizations can remo
 For non-sensitive values, you can add them directly to the appropriate .env file for each mode.
 
 For sensitive values, if building with github actions we recommend using [github secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) and setting them in the [build action environment](https://docs.github.com/en/actions/learn-github-actions/variables).
-You can see an example of how these are used in our [Firebase Live / Preview actions.](https://github.com/jwplayer/ott-web-app/blob/develop/.github/workflows/firebase-live.yml#L14) 
+You can see an example of how these are used in our [Firebase Live / Preview actions.](https://github.com/jwplayer/ott-web-app/blob/develop/.github/workflows/firebase-live.yml#L14)
 
-If building manually, you can create an .env.[mode].local file and add the values there. These files are git ignored which will prevent leaking your secrets to version control. 
+If building manually, you can create an .env.[mode].local file and add the values there. These files are git ignored which will prevent leaking your secrets to version control.
 
-> Note: env variables must begin with the 'APP_' prefix or they are ignored by our vite configuration.
+> Note: env variables must begin with the 'APP\_' prefix or they are ignored by our vite configuration.
```

Comment:
Hm, do we need to change `APP_`?

Reply:
It's the auto formatting, when I save it modifies it 😨

Reply:
The preview seems to ignore the `\` so maybe it's just an auto-formatting behaviour